packages/transport/src/core/backbone/RESTClientAuthenticate.ts

```diff
@@ -68,7 +68,7 @@ export class RESTClientAuthenticate extends RESTClient {
         });
     }
 
-    public override async put<T>(path: string, data: any, config: AxiosRequestConfig = {}): Promise<ClientResult<T>> {
+    public override async put<T>(path: string, data: any = {}, config: AxiosRequestConfig = {}): Promise<ClientResult<T>> {
```

Comment:
I'd vote for undoing this

Reply:
why?

Reply:
b/c defaults here can lead to unexpected behaviour.

Reply:
just pass {} if you **really** want to send an empty object